.github/workflows/capistrano.yml

```
@@ -75,17 +80,48 @@ jobs:
+    #       export GEM_PATH="$USER_GEM_HOME:$(ruby -rrubygems -e 'print Gem.path.join(\":\")')"
+    #       export PATH="$USER_GEM_BIN:$PATH"
+    #       ruby -v
+    #       ruby -S gem install bundler -v 1.17.3 --user-install --bindir "$USER_GEM_BIN" --no-document
+    #       bundle _1.17.3_ install --path vendor/bundle --without development test --jobs 4 --retry 3
+    #   - name: Deploy with Capistrano
+    #     run: bundle exec cap anonymized deploy
     if: inputs.environment == 'anonymized'
-    runs-on: ubuntu-22.04
+    runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
```

actionlint (1.7.12)

[error] the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)